backend/lcfs/web/api/organization/views.py

```diff
@@ -28,6 +32,65 @@
 get_async_db = dependencies.get_async_db_session
 
 
+@router.get("/export", response_class=StreamingResponse)
+@roles_required("Government")
+async def export(request: Request, db: AsyncSession = Depends(get_async_db)):
+    """
+    Endpoint to export information of all organization
```

Comment:
expand on this doc string to include what file format is and a brief on the spreadsheet builder

Reply:
Of course!